### DISCUSSION TITLE
append non-graph data items on Data object

### DISCUSSION THREAD

**Discussion by HansLeonardVanBrueggemann** [pyg-team/pytorch_geometric#3571] [Q&A]
Hi there 

DISCLAIMER im new to pytorch geometric. That said, 

I have a dataset with is composed by graphs + a small set of dense features, a feature vector, if you will, associated with each graph + an output class Y. 

So each data point x = { graph = {edges, feature_nodes}, feature_vec, y }

how can I include that feature_vec in my torch_geometric.data.Data object so that, when I used the loader, I will be able to retrieve both graph, feature vector and y? 

I was thinking of extending the Data class, but im not sure if that is the way to go.

**Comment by rusty1s** [comment]
You should be able to simply add it as an attribute to `data`:
```python
data = Data(x=x, edge_index=edge_index, feature_vec=feature_vec, y=y)
```
